src/Widget/Translation/Service/FilterForKeyWithFallback.php

```diff
@@ -21,7 +21,14 @@ public function __invoke(array $values, $preferredLanguage, $mainLanguage = 'nl'
         }
 
         if (!$this->hasPreferred($values, $preferredLanguage)) {
-            return isset($values[$this->fallBackLanguage]) ? $values[$this->fallBackLanguage] : $values[$mainLanguage];
+            if (isset($values[$this->fallBackLanguage])) {
+                return $values[$this->fallBackLanguage];
+            } elseif (isset($values[$mainLanguage])) {
+                return $values[$mainLanguage];
+            } else {
+                $firstKey = array_keys($values)[0];
+                return $values[$firstKey];
+            }
```

Comment:
You don't need IF-ELSE-IF since every IF has a return statement, you can instead have
```
if (//..) {
return something;
}
if (//..) {
return something;
}
```
which is easier to follow when reading the code